packages/contracts-bedrock/test/L1/ETHLockbox.t.sol

```diff
@@ -643,9 +643,9 @@ contract ETHLockbox_MigrateLiquidity_Test is ETHLockbox_TestInit {
     }
 }
 
-/// @title ETHLockbox_Unclassified_Test
-/// @notice Contains unclassified tests related to ETHLockbox.
-contract ETHLockbox_Unclassified_Test is ETHLockbox_TestInit {
+/// @title ETHLockbox_Uncategorized_Test
+/// @notice Contains uncategorized tests related to ETHLockbox.
+contract ETHLockbox_Uncategorized_Test is ETHLockbox_TestInit {
```

Comment:
Why is this uncategorized instead of something like `ETHLockbox_ProxyAdminOwner_Test`? (It may be that I'm unfamiliar with the new convention here)